tests/system/data_sources/test_bigquery.py

```diff
@@ -1154,9 +1155,10 @@ def test_custom_query():
     "data_validation.state_manager.StateManager.get_connection_config",
     return_value=BQ_CONN,
 )
-def test_bigquery_generate_table_partitions(mock_conn):
-    """Test generate table partitions on BigQuery"""
-    generate_partitions_test(EXPECTED_PARTITION_FILTER)
+def test_generate_partitions(mock_conn, tmp_path):
```

Comment:
Where is `tmp_path` coming from? I looked at all variables available at the top and it's not there, I'm super curious here haha same question applies to the other systems too